packages/antd/package.json

```diff
@@ -37,7 +37,7 @@
     "@ant-design/icons": "^4.0.0",
     "@rjsf/core": "^5.12.x",
     "@rjsf/utils": "^5.12.x",
-    "antd": "^4.0.0",
+    "antd": "^4.24.0",
```

Comment:
Shouldn't this be `"^4.24.0 || ^5.8.5" ? Otherwise antd v5 users will get a peer dependency warning/error

Reply:
the caret symbol will handle it right?.. npm installs latest version of v4..

Reply:
Ah okay.. Understood..

Reply:
Fixed